apps/web/screens/Adgerdir/Home.tsx

```diff
@@ -71,6 +71,13 @@ const Home: Screen<HomeProps> = ({ frontpage, pages, tags, namespace }) => {
 
   return (
     <>
+      <HeadWithSocialSharing
+        title={`Viðspyrna fyrir Ísland`}
+        description={frontpage.description}
+        imageUrl={frontpage.featuredImage?.url}
+        imageWidth={frontpage.featuredImage?.width.toString()}
+        imageHeight={frontpage.featuredImage?.height.toString()}
```

Comment:
```suggestion
        imageWidth={frontpage.featuredImage?.width?.toString()}
        imageHeight={frontpage.featuredImage?.height?.toString()}
```
To prevent toString of undefined